core/transformations_defines.h

```diff
@@ -26,16 +26,16 @@
 #ifndef XMIPPCORE_CORE_TRANSFORMATIONS_DEFINES_H_
 #define XMIPPCORE_CORE_TRANSFORMATIONS_DEFINES_H_
 
-#define NEAREST 0
-#define LINEAR 1
-#define BSPLINE2 2
-#define BSPLINE3 3
-#define BSPLINE4 4
 
-#define IS_INV true
-#define IS_NOT_INV false
-#define DONT_WRAP false
-#define WRAP true
+namespace xmipp_transformation
+{
+	enum XmippInterpolation {NEAREST=0, LINEAR=1, BSPLINE2=2, BSPLINE3=3, BSPLINE4=4}; // Interpolation type
```

Comment:
please use enum class (see e.g. [this link](https://stackoverflow.com/a/18335862))

Reply:
If I use class, the constants would expand to xmipp_transformation::XmippInterpolation::NEAREST. Without the class, we remove at least one level, and there is no ambiguity, anyway.

Reply:
I think it follows the name convention: both branches are called co_libsvm

Reply:
enum class are preferred because they also cannot be cast to e.g. int
Also, you can skip the namespace then